build.gradle

```diff
@@ -80,22 +113,22 @@ task processCmuDict {
                         })
                         def transcriptionSyllabified = allophoneSet.syllabify(transcriptionMapped)
                         def transcriptionNoSpaces = transcriptionSyllabified.replaceAll(' ', '')
-                        dest.println "$lemma $transcriptionNoSpaces"
+                        dest.writeLine("$lemma $transcriptionNoSpaces")
                     }
                 }
             }
         }
     }
 }
 
-compileLexicon {
-    dependsOn processCmuDict
-    allophonesFile = file('modules/en/us/lexicon/allophones.en_US.xml')
-    lexiconFile = processCmuDict.destFile
+tasks.named('compileLexicon').configure {
+    dependsOn processCmuDictTask
+    allophonesFile = allophoneSetFile.get().asFile
+    lexiconFile = processCmuDict.destFile.get().asFile
```

Comment:
This cannot be streamlined until marytts/gradle-marytts-lexicon-compiler-plugin#10 is resolved

Reply:
Resolved in 7a49084b5792d6bacdbc342e1c08b34015285234